dashboard/app/models/odometer.rb

```diff
@@ -0,0 +1,6 @@
+class Odometer < Widget
+
+  before_validation do
+    self.href = 'odometer/odometer.html'
```

Comment:
was there a particular reason you did it this way instead of just overriding the href method? Does overriding SerializedProperties methods not work?